testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCompression.java

```diff
@@ -92,12 +92,11 @@ public void testSnappyCompressedParquetTableCreatedInTrino()
     @Test(groups = HIVE_COMPRESSION)
     public void testSnappyCompressedParquetTableCreatedInTrinoWithNativeWriter()
     {
-        // TODO (https://github.com/trinodb/trino/issues/6377) Parquet files written by native Parquet writer cannot be read by Hive
+        // TODO (https://github.com/trinodb/trino/issues/7953) Native Parquet Writer writes Parquet V2 files that are not yet adopted by other engines
         assertThatThrownBy(() -> testSnappyCompressedParquetTableCreatedInTrino(true))
-                .hasStackTraceContaining("at org.apache.hive.jdbc.HiveQueryResultSet.next") // comes via Hive JDBC
+                .hasStackTraceContaining("at org.apache.hive.jdbc.Utils.verifySuccess") // comes via Hive JDBC
                 .extracting(Throwable::toString, InstanceOfAssertFactories.STRING)
-                // CDH5's Parquet uses parquet.* packages, while e.g. HDP 3.1's uses org.apache.parquet.* packages.
-                .matches("\\Qio.trino.tempto.query.QueryExecutionException: java.sql.SQLException: java.io.IOException:\\E (org.apache.)?\\Qparquet.io.ParquetDecodingException: Cannot read data due to PARQUET-246: to read safely, set parquet.split.files to false");
+                .matches("\\Qio.trino.tempto.query.QueryExecutionException: java.sql.SQLException: java.io.IOException:\\E ((org.apache.)?parquet.io.ParquetDecodingException: Can not read value at 0 in block -1 in file .*|org.apache.hadoop.hive.ql.metadata.HiveException: java.lang.ClassCastException: org.apache.hadoop.io.BytesWritable cannot be cast to org.apache.hadoop.hive.serde2.io.HiveVarcharWritable)");
```

Comment:
add a comment when "Can not read value at 0 in block -1" is to be expected, and when "BytesWritable cannot be cast". Does this depend on Hive version?